EventFilter/RawDataCollector/src/RawDataCollectorByLabel.cc

```diff
@@ -71,10 +71,10 @@ void RawDataCollectorByLabel::produce(Event & e, const EventSetup& c){
        if(verbose_ > 1) std::cout << "Copying data from FED #" << j << std::endl;
        FEDRawData & fedDataProd = producedData->FEDData(j);
        if ( fedDataProd.size() != 0 ) {
-	 if(verbose_ > 1) {
-	   std::cout << " More than one FEDRawDataCollection with data in FED ";
-	   std::cout << j << " Skipping the 2nd\n";
-	 }
+        if(verbose_ > 1) {
```

Comment:
@icali general comment here and in any other place: why do you use cout instead of the MessageLogger? A LogInfo would seem appropriate to me

Reply:
IMHO, this file should not need to be changed at all. The only difference are the whitespaces.

Reply:
probably originally was simply an indentation issue than triggered the modification. I will resolve this conversation considering that no changes are then needed.